util/state_division_to_dcid.py

```diff
@@ -0,0 +1,167 @@
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Geo utility that maps the names of reporting areas in NNDSS to Data Commons
+place nodes.
+"""
+
+_PLACE_MAP = {
+    "U.S. Residents": "country/USA",
+    "U.S. Residents, excluding U.S. Territories": "country/USA",
+    "United States": "country/USA",
+    "New England": "usc/NewEnglandDivision",
+    "Connecticut": "geoId/09",
+    "Maine": "geoId/23",
+    "Massachusetts": "geoId/25",
+    "New Hampshire": "geoId/33",
+    "Rhode Island": "geoId/44",
+    "Vermont": "geoId/50",
+    "Middle Atlantic": "usc/MiddleAtlanticDivision",
+    "New Jersey": "geoId/34",
+    "New York": "geoId/36",
+    "New York (excluding New York City)": "",
+    "New York City": "",
+    "Pennsylvania": "geoId/42",
+    "East North Central": "usc/EastNorthCentralDivision",
+    "Illinois": "geoId/17",
+    "Indiana": "geoId/18",
+    "Michigan": "geoId/26",
+    "Ohio": "geoId/39",
+    "Wisconsin": "geoId/55",
+    "West North Central": "usc/WestNorthCentralDivision",
+    "Iowa": "geoId/19",
+    "Kansas": "geoId/20",
+    "Minnesota": "geoId/27",
+    "Missouri": "geoId/29",
+    "Nebraska": "geoId/31",
+    "North Dakota": "geoId/38",
+    "South Dakota": "geoId/46",
+    "South Atlantic": "usc/SouthAtlanticDivision",
+    "Delaware": "geoId/10",
+    "District of Columbia": "geoId/11",
+    "Washington DC": "geoId/11001",
+    "Florida": "geoId/12",
+    "Georgia": "geoId/13",
+    "Maryland": "geoId/24",
+    "North Carolina": "geoId/37",
+    "South Carolina": "geoId/45",
+    "Virginia": "geoId/51",
+    "West Virginia": "geoId/54",
+    "East South Central": "usc/EastSouthCentralDivision",
+    "Alabama": "geoId/01",
+    "Kentucky": "geoId/21",
+    "Mississippi": "geoId/28",
+    "Tennessee": "geoId/47",
+    "West South Central": "usc/WestSouthCentralDivision",
+    "Arkansas": "geoId/05",
+    "Louisiana": "geoId/22",
+    "Oklahoma": "geoId/40",
+    "Texas": "geoId/48",
+    "Mountain": "usc/MountainDivision",
+    "Arizona": "geoId/04",
+    "Colorado": "geoId/08",
+    "Idaho": "geoId/16",
+    "Montana": "geoId/30",
+    "Nevada": "geoId/32",
+    "New Mexico": "geoId/35",
+    "Utah": "geoId/49",
+    "Wyoming": "geoId/56",
+    "Pacific": "usc/PacificDivision",
+    "Alaska": "geoId/02",
+    "California": "geoId/06",
+    "Hawaii": "geoId/15",
+    "Oregon": "geoId/41",
+    "Washington": "geoId/53",
+    "U.S. Territories": "",
+    "American Samoa": "geoId/60",
+    "Commonwealth of Northern Mariana Islands": "geoId/69",
+    "Guam": "geoId/66",
+    "Puerto Rico": "geoId/72",
+    "U.S. Virgin Islands": "geoId/78",
+    "US Virgin Islands": "geoId/78",
+    "Non-U.S. Residents": "country/USA",
+    "Total": "",
```

Comment:
Can remove such invalid places if the function return empty sting for missing keys.

Reply:
Done.